modmesh/app/euler1d.py

```diff
@@ -61,73 +66,315 @@ def load_app():
 class QuantityLine:
     ana: matplotlib.lines.Line2D = None
     num: matplotlib.lines.Line2D = None
+    axis: matplotlib.pyplot.axis = None
     name: str = ""
     unit: str = ""
 
+    def bind_axis(self, axis):
+        self.axis = axis
+
     def update(self, adata, ndata):
         self.ana.set_ydata(adata.copy())
         self.num.set_ydata(ndata.copy())
+        self.axis.relim()
+        self.axis.autoscale_view()
```

Comment:
Sometimes we do not want the plot to autoscale.  It's OK to use it now but we should plan for adding an option to control the behavior in the future.